Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -135,8 +137,22 @@ private void addSmartDashboardWidgets( )
     SmartDashboard.putData("AutoDrivePathForward2", new AutoDrivePath(m_swerve, "forward2m", true));
     SmartDashboard.putData("AutoDrivePathBackward2", new AutoDrivePath(m_swerve, "backward2m", true));
 
-    // On-the-fly path generation buttons
-    SmartDashboard.putData("ResetOdometryToLimelight", new ResetOdometryToLimelight(m_swerve, m_vision, 0));
+    SmartDashboard.putData("ArmSetHeightStow", new ArmSetHeightStow(m_elbow, m_wrist));
```

Comment:
Not quite sure why these show as being added by you, because I think they were in my PR that created the shuffleboard layout.